src/Symfony/Component/OptionsResolver/OptionsResolver.php

```diff
@@ -348,6 +354,52 @@ public function getDefinedOptions()
         return array_keys($this->defined);
     }
 
+    /**
+     * Deprecates an option, allowed types or values.
+     *
+     * Instead of passing the message, you may also pass a closures with the
```

Comment:
typo: no s for closure

Reply:
Fixed, but there are still 11 occurrences of this typo here (should be fixed in 2.8).